### ISSUE TITLE
Links to some project Notes on user Profile page are broken, the target location is unexpected

### ISSUE THREAD

**Issue by yshish** [zooniverse/Panoptes-Front-End#4015]
## Expected behavior
When visiting a profile page (Recent comments) and looking at any comments posts in Notes on AmazonCam Tambopata, they should show what project they belong to. They don't.
When I click on one, it should lead me directly to the post in the project Notes. It doesn't.
Also, when I hover over the link, it should display where it takes me. It doesn't either.

## Current behavior
When I visit my profile page, all my comments in Notes of another project (WesternShield) display the project name and not just 'Notes' unlike the AmazonCam. It shows them as Notes only. 
Then, when I click on any expecting to get to the discussion where it comes from, it takes me to the Zooniverse Home page instead. So there is no way to get to my AmazonCam Notes posts from my Profile page. I have to search them by their ID but there is no way to see what project they are related to.
Hovering over the link shows their target location as my Profile page although it actually takes me to the Zooniverse Home page. So it doesn't correspond either.

## Steps to replicate
Visit my profile page or a profile page of any other volunteer commenting on various Panoptes projects in Notes. Two examples: 
me: https://www.zooniverse.org/users/yshish
random user: https://www.zooniverse.org/users/MvGulik
Let me know if a screenshot is needed.

## Additional information
Looking at posts from the random user, https://www.zooniverse.org/users/MvGulik , there are both correctly and wrongly displayed Notes posts. The broken ones don't seem to come from AmazonCam as far as I can tell (but since it doesn't work, I can't make sure what project they come from). So it is probably not only about AmazonCam project vs. Western Shield. 

Hope it makes sense.

- **Operating system:** Linux x86_64
- **Browser:** Chromium 58

**Comment by eatyourgreens** [comment]
I've experienced this same bug, but only intermittently. For whoever picks this up, it seems like the project information to build the links is not always returned by the API query.

**Comment by yshish** [comment]
Ah, true! I've just tried some other volunteers from AmazonCam and to my surprise, their Notes posts did display properly. So I went back to my profile page and although the links first displayed wrongly, suddenly they re-appeared as correct ones. 
But going again to MvGulik's profile page showed me the broken links again as before, so I did make a screenshot of the cursor hovering above the wrongly displayed link showing it will take me to MvGulik's profile page and not to the post itself:

![ysh_screenshot_2017-08-30_21-17-18_hoverover-link](https://user-images.githubusercontent.com/7211888/29891018-ee6d5ce8-8dc9-11e7-9f93-6abde04d4a1c.png)
